packages/ui/src/layout/DashboardLayout.tsx

```diff
@@ -1,270 +1,20 @@
-import { Fragment, ReactNode, useState } from "react";
-import { Dialog, Transition } from "@headlessui/react";
-import {
-    Bars3Icon,
-    BellIcon,
-    Cog6ToothIcon,
-    FlagIcon,
-    FolderIcon,
-    HomeIcon,
-    XMarkIcon,
-} from "@heroicons/react/24/outline";
-import { MagnifyingGlassIcon } from "@heroicons/react/20/solid";
-import logo from "../images/logo.png";
-import { classNames } from "../helpers/classHelper";
-import { useLocation } from "react-router-dom";
+import { Outlet } from "react-router-dom";
 
-export const DashboardLayout: React.FC<{ children: ReactNode }> = (props) => {
-    const [sidebarOpen, setSidebarOpen] = useState(false);
-    const location = useLocation();
-
-    const navigation = [
-        { name: "Dashboard", href: "/dashboard", icon: HomeIcon },
-        { name: "Flags", href: "/flags", icon: FlagIcon },
-        { name: "Segments", href: "/segments", icon: FolderIcon },
-    ];
+import Sidebar from "../components/shared/Sidebar";
 
+const DashboardLayout = () => {
     return (
-        <div className="max-w-7xl mx-auto">
-            {/* Sidebar for mobile component, swap this element with another sidebar if you like */}
```

Comment:
I like to have a separate component for the sidebar, but your changes would break the mobile view. You can use the same approach implemented here in the new component for the sidebar.